docs/building-spark.md

```diff
@@ -123,6 +123,21 @@ To produce a Spark package compiled with Scala 2.10, use the `-Dscala-2.10` prop
     ./dev/change-scala-version.sh 2.10
     mvn -Pyarn -Phadoop-2.4 -Dscala-2.10 -DskipTests clean package
 
+# PySpark Tests with Maven
+
+If you are building PySpark and wish to run the PySpark tests you will need to build an assembly JAR and also ensure you have built with hive support.
+
+{% highlight bash %}
+build/mvn -Pyarn -Phadoop-2.4 -Dhadoop.version=2.4.0 -DskipTests clean package -Phive -Phive-thriftserver
+./python/run-tests
+{% endhighlight %}
+
+The run-tests script also can be limited to a specific Python version or a specific module
+
+    ./python/run-tests --python-executables=python --modules=pyspark-sql
+
+**Note:** You can also run Python tests with an sbt build, provided your assembly JAR includes hive support.
```

Comment:
Again, we should remove the reference to assembly jar from here, something like 'provided you build spark with hive support' should suffice.